### PR TITLE
Pixbuf doc #13

### PR DESCRIPTION
**Pull request by gbersac**

### CODE REVIEW

File: src/pixbuf.rs [hunk 1]
```diff
@@ -25,12 +25,32 @@ glib_wrapper! {
 }
 
 impl Pixbuf {
+
```
Comment (GuillaumeGomez) [inline src/pixbuf.rs]: This empty line is unneeded.

File: src/pixbuf.rs [hunk 2]
```diff
@@ -25,12 +25,32 @@ glib_wrapper! {
...
+    /// Create a new `Pixbuf` initialized with the bits of `vec`
+    ///
+    /// `colorspace`: flag for the type of color channel. Only possible value is
+    /// `0` for RGB.
+    ///
+    /// `has_alpha`: true if there is one alpha canal
+    ///
+    /// `bits_per_sample`: must be `8`.
+    ///
+    /// `row_stride`: distance in bytes between row starts. Usually the number of
```
Comment (GuillaumeGomez) [inline src/pixbuf.rs]: "multiply" -> "multiplied".

I also think the "Usually" is useless here.

**Comment by GuillaumeGomez** [comment]
Thanks for your work! Please fix the nits and it's good for me. cc @gkoz

**Comment by gbersac** [comment]
I fixed it. If this pull request is a problem for your doc generator as mentioned by @gkoz [here](https://github.com/gtk-rs/gdk-pixbuf/issues/13#issuecomment-198819135), don't hesitate to drop it.

**Comment by GuillaumeGomez** [comment]
Don't worry, we'll explain the needed changes if needed.

**Pull request closed without merging**